From 7b69ee43eec26dac91ef7c3ed0a4578b74b4ab30 Mon Sep 17 00:00:00 2001
From: Liu Ying <Ying.Liu@freescale.com>
Date: Thu, 24 Sep 2015 11:16:18 +0800
Subject: [PATCH 0577/5242] MLK-11613 ARM: dts: imx6qp-sabresd: Enable
 prefetch engines

commit  7876d01f22e8b9743cb42fbde7def5ea22a15c58 from
https://source.codeaurora.org/external/imx/linux-imx.git

This patch enables the four prefetch engines on the imx6qp-sabresd platform.

Signed-off-by: Liu Ying <Ying.Liu@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6qp-sabresd.dts |   16 ++++++++++++++++
 1 file changed, 16 insertions(+)

diff --git a/arch/arm/boot/dts/imx6qp-sabresd.dts b/arch/arm/boot/dts/imx6qp-sabresd.dts
index f1b9cb1..3038ccb 100644
--- a/arch/arm/boot/dts/imx6qp-sabresd.dts
+++ b/arch/arm/boot/dts/imx6qp-sabresd.dts
@@ -53,3 +53,19 @@
 &pcie {
 	status = "disabled";
 };
+
+&pre1 {
+	status = "okay";
+};
+
+&pre2 {
+	status = "okay";
+};
+
+&pre3 {
+	status = "okay";
+};
+
+&pre4 {
+	status = "okay";
+};
-- 
1.7.9.5

